### PR TITLE
Demonstrate inheritance on a virtual test example

### PR DESCRIPTION
**Pull request by psss**
In order to help with questions like #1124.

### CODE REVIEW

File: docs/examples.rst [hunk 1]
```diff
@@ -190,6 +190,36 @@ L1 Metadata Specification::
     pass directory path must be defined
 
 
+Inherit Tests
```
Comment (lukaszachy) [inline docs/examples.rst:193]: From words 'Inherit Tests' I'd assume using same set of tests across multiple plans.

How about "Share common attributes"?

**Review by lukaszachy — COMMENTED** [review]
Could be good to have https://github.com/teemtee/tmt/issues/1124#issuecomment-1144536105 as example somewhere too, IMHO it is more readable (for me).

File: docs/examples.rst (hunk 1 above)
Comment (FrNecas) [reply docs/examples.rst:193]: Agreed, maybe even "Share common test attributes"? Though I am not sure if that isn't too long and verbose.

**Review by ukulekek — APPROVED** [review]
Given the comments above, it looks good

File: docs/examples.rst (hunk 1 above)
Comment (psss) [reply docs/examples.rst:193]: "Share common attributes" sounds good!

File: docs/examples.rst [hunk 2]
```diff
@@ -190,6 +190,36 @@ L1 Metadata Specification::
...
+
+Sometimes it might be useful to reuse test code by providing
+different parameter to the same test script. In such cases
+inheritance allows to easily share the common setup::
+
+    test: ./test.sh
+    require: curl
+
+    /fast:
+        description: Check download options (quick smoke test)
```
[the remark this answers is not in the document]
Comment (psss) [reply docs/examples.rst:204]: Make sense, shortened.

**Review by lukaszachy — APPROVED** [review]

**Pull request merged**